client/src/components/Sub_Questions/QuestionSearch.jsx

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
 
-export default function QuestionsSearch() {
+export default function QuestionsSearch({ questionList, setFilteredList }) {
+  const handleChange = (e) => {
+    if (e.target.value.length >= 3) {
+      const newList = questionList.filter((question) => question.question_body
+        .toLowerCase()
+        .includes(e.target.value.toLowerCase()));
+      setFilteredList(newList);
```

Comment:
looks here like you're implementing a live search on user typing. looks good to me. although i've been having a lot of issues with using an additional state to store filtered data so i'm curious to see how your implementation would differ than mine.

Reply:
not sure how you implemented but this is the only way I know for now so I would love to learn how you did it and compare them :)